### ISSUE TITLE
bug: exec-env does not work when invoked via asdf direnv plugin

### ISSUE THREAD

**Issue by log2** [jthegedus/asdf-gcloud#65]
### Describe the bug

During environment caching, asdf direnv plugin invokes exec-env on all plugins registered in .tool-versions; unfortunately, exec-env is not called via absolute path, so calling gcloud plugin's exec-env results in:

```
direnv: loading ~/.asdf/plugins/gcloud/bin/exec-env
./exec-env:10: ./lib/utils.bash: No such file or directory
```

Since that script internally uses dirname and path used to invoke script is relative to current directory (thus, `./exec-env`).

Using `realpath` around `"$current_script_path"`, if available, should solve this issue.

### Steps to reproduce

1. Have `.envrc` in place
2. Add `use asdf` in `.envrc`, if missing
3. Ensure that a version for gcloud is declared in asdf, either via `.tool-versions` or using `asdf shell gcloud`
4. Enter `direnv allow`

### asdf version

v0.10.2

### asdf-gcloud version

main f59e332

### gcloud version

394.0.0

**Comment by jthegedus** [comment]
This might be fixed by #63, please check with `v1.2.1`

**Comment by log2** [comment]
Thank you @jthegedus, you are right: the present issue is fixed with that version.

Closing issue.
